numbers/logSpace.go

```diff
@@ -7,7 +7,23 @@ import (
 
 //these two values mark the overflow/underflow boundaries for math.Exp(x)
 func LogCanConvert(x float64) bool {
-	return x > 7.09782712893383973096e+02 || x < -7.45133219101941108420e+02
+	return x > 709.782712893383973096e+02 || x < -745.133219101941108420e+02
```

Comment:
Is this change correct?  Those original values look correct to me.  Maybe you didn't notice the "e+02" at the end?
If I am thinking about this correctly, I think you actually want the negation of this: x < 700 && x > -700
I might be mixed up, but something to double-check

Reply:
Oh yes, these were correct beforehand. I'll adjust